service/controller/v19/templates/cloudformation/guest/record_sets.go

```diff
@@ -20,12 +20,13 @@ const RecordSets = `{{define "record_sets"}}
   EtcdRecordSet:
     Type: AWS::Route53::RecordSet
     Properties:
+      AliasTarget:
+        DNSName: !GetAtt ApiLoadBalancer.DNSName
+        HostedZoneId: !GetAtt ApiLoadBalancer.CanonicalHostedZoneNameID
+        EvaluateTargetHealth: false
       Name: 'etcd.{{ $v.ClusterID }}.k8s.{{ $v.BaseDomain }}.'
       HostedZoneId: !Ref 'HostedZone'
-      TTL: '60'
-      Type: CNAME
-      ResourceRecords:
-        - !GetAtt {{ $v.MasterInstanceResourceName }}.PrivateDnsName
```

Comment:
I miss how we add the master to the LB backend.

Reply:
This is the API loadbalancer. So this should already be there.

Reply:
https://github.com/giantswarm/aws-operator/blob/master/service/controller/v19/templates/cloudformation/guest/load_balancers.go#L19